chia/plotting/manager.py

```diff
@@ -170,6 +170,13 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._lock.release()
 
+    def reset(self):
+        self.last_refresh_time = time.time()
```

Comment:
I would have expected this to set `last_refresh_time` to 0. Setting it to the current timestamp will make it not trigger again for a while, right?